higlass/server.py

```diff
@@ -339,8 +364,8 @@ class Server:
         StringIO stream in memory.
     root_api_address: str, optional
         Root path of the reported api_address, instead of http://{host}:{port}.
-        Can be used for redirecting clients to a proxy. Can include {host} and {port}
-        to use these variables.
+        Can be used for redirecting clients to a proxy. Can include {host}, {port}
+        and {unix_filename} to use these variables.
```

Comment:
So `root_api_address` could be any unix filename?

Reply:
Yeah. This is usually marked with a different schema, like `http+unix://`

Reply:
`http_unix://`? Isn't it just `unix://`?

Reply:
oh, I see, the API address returned to the client is `http+unix` but the location specified is `unix://`.